### PR TITLE
Method to install procedures, subscribe to multiple tables, etc.

### PR DESCRIPTION
**Pull request by epoberezkin**
Please let me know if this is too much of a change. It breaks backward compatibility as constructor expects options instead of client.
Changes
1. Method to install procedures - I think it is more convenient as it allows to update them without the need of library users to manage it in db. Also it allows to make them configurable via options.
2. Subscribe to multiple table with one call
3. All tables send notifications via the same channel - no need to have multiple subscriptions to postgres

TODO
1. Optionally do not send update notification when there were no changes 
2. Configure the list of columns included in payload, possibly separately for data and old_data (in case there are some big columns that may not be needed in updates)

### CODE REVIEW

File: install.sql [hunk 1]
```diff
@@ -1,51 +1,44 @@
...
-    _json_ := '{"table": "' || TG_TABLE_NAME || '",
-      "operation": "delete",
-      "timestamp": "' || CURRENT_TIMESTAMP || '",
-      "data": ' || row_to_json(OLD) || '}';
+  data := '{"table": "' || TG_TABLE_NAME || '",
+      "operation": "' || TG_OP || '",
+      "timestamp": "' || CURRENT_TIMESTAMP || '"';
+
+  IF TG_OP = 'INSERT' OR TG_OP = 'UPDATE' THEN
+    data := data || ', "data": ' || row_to_json(NEW);
```
Comment (paulserraino) [inline install.sql]: I like this :+1:

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pg-dispatch",
-  "version": "0.0.1",
+  "version": "0.1.0",
```
Comment (paulserraino) [inline package.json]: I think this would better  `"version": "0.0.2"`

Comment (epoberezkin) [inline package.json]: yes, no problem

**Comment by epoberezkin** [comment]
fixing tests now...

**Comment by paulserraino** [comment]
awesome, ha. I just about to ask

File: index.js [hunk 3]
```diff
@@ -1,38 +1,109 @@
...
+        notifyTriggerFunc: 'data_change_notify_trigger_func',
+        addNotifyTrigger: 'add_notify_trigger_to_table',
+        removeNotifyTrigger: 'remove_notify_trigger_to_table'
+    },
+    operationEvents: true
+};
+
+function PGObserver (opts) {
+    if (!(this instanceof PGObserver)) return new PGObserver(opts);
+    this.client = opts.client || new pg.Client(opts.conString);
```
Comment (paulserraino) [inline index.js]: ah, this is breaking the subscribe test. The first argument in the test script was originally just pg client.
It should now be initialized as  `self.pgevents = PGEvents({client: client})`
https://github.com/paulserraino/pg-pubsub/blob/master/tests/tests.js#L10

**Comment by epoberezkin** [comment]
yes, I've fixed them all... I need to leave for about an hour, will commit all fixes when I come back.

**Comment by epoberezkin** [comment]
actually here they are

**Comment by paulserraino** [comment]
awesome, thanks for the PR! :+1:

**Pull request merged**